admin-dev/themes/new-theme/js/pages/currency/form/components/CurrencyFormatForm.vue

```diff
@@ -0,0 +1,112 @@
+<!--**
+ * 2007-2019 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
```

Comment:
put these files into
`admin-dev/themes/new-theme/js/pages/currency/form/components/currency/FormatForm.vue`
otherwise in few weeks / months we will have 200 files in components.

Reply:
you're right  :sweat_smile: 
Forget they mix components & pages, which was a really bad idea :(